uk.co.compucorp.civicrm.hrleaveandabsences/xml/schema/CRM/HRLeaveAndAbsences/LeaveRequestCalendarFeedConfig.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<table>
+  <base>CRM/HRLeaveAndAbsences</base>
+  <class>LeaveRequestCalendarFeedConfig</class>
```

Comment:
Is this really a config or will each instance of this be a Feed itself? Will a Feed contain any additional information that is not in the Config already?

Reply:
Its a config actually. The feed is the actual leave requests data generated off the configurations set for it.